### PR TITLE
Added jupyter notebook instructions

### PR DESCRIPTION
**Pull request by mkphuthi**

### CODE REVIEW

File: getting_started/jupyter_notebooks.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+where `c022` is the hostname of the worker node. Copy it.
+
+3. Start up VS Code and choose the server you just started. Do this by opening the command palette (usually `Cmd+Shift+P`) and search for `Jupyter: Select Jupyter Server for Connections`. Enter the url from step 2.
+
+4. Open your notebook and write your code
+
+> Note 1: VS Code does not automatically save notebooks so you will have to keep saving your work. If your wall time runs out before you save, you will have to restart the server and save again.
+
+> Note 2: Running the notebook as above is really slow. If your use case uses small amounts of data and does not require special hardware, you might be better off mounting your Arjuna home directory locally on your machine using something like [FUSE](https://susanqq.github.io/jekyll/pixyll/2017/09/05/remotefiles/) and running your server locally.
```
Comment (awadell1) [inline getting_started/jupyter_notebooks.md:45]: Cut this. A) we don't have docs for it and b) rclone is probably the better/ lower impact move

File: getting_started/jupyter_notebooks.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Instructions
+1. Allocate an interactive worker node with the resources you need, for example:
+```
+srun -N 1 -n 1 -A <account> -p <partition> --mem=2G -t 01:00:00 --pty bash
+```
+or if you need GPUs:
+```
+srun -N 1 -n 1 -A <account> -p gpu --gres=gpu:1 --mem=2G -t 01:00:00 --pty bash
+```
+with the appropriate account and partition. Be reasonable about your needs. If you are not running any parallelized code, you will need 2 cores at most, if you don't need a GPU, don't request it. The time requested should be a reflection of how long you will actually work on the node.
```
[the remark this answers is not in the document]
Comment (awadell1) [reply getting_started/jupyter_notebooks.md:26]: It has to be `-N 1` right? If so add a callout like this:

```
> Jupyter can only run on a single machine. Be sure to specify `-N 1`

```

**Review by awadell1 — CHANGES_REQUESTED** [review]
Thanks for the docs! Just some editorial tweaks

File: getting_started/jupyter_notebooks.md (hunk 1 above)
Comment (mkphuthi) [reply getting_started/jupyter_notebooks.md:45]: A basically says save your work frequentely, not entirely sure what docs are needed.

Comment (mkphuthi) [reply getting_started/jupyter_notebooks.md:45]: Actually, turns out you don't need the server to be running to save the notebook.

File: getting_started/jupyter_notebooks.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+```
+srun -N 1 -n 1 -A <account> -p gpu --gres=gpu:1 --mem=2G -t 01:00:00 --pty bash
+```
+with the appropriate account and [partition](../about/hardware#partitions). 
+
+> Be reasonable about your needs. If you are not running any parallelized code, you will need 2 cores at most, if you don't need a GPU, don't request it. The time requested should be a reflection of how long you will actually work on the node.
+
+> Jupyter can only run on a single machine. Be sure to specify `-N 1`
+
+2. In the worker node, activate the environment with jupyter installed that you want to use and start the notebook server e.g. in conda:
```
Comment (awadell1) [inline getting_started/jupyter_notebooks.md:38]: Change to: `On the worker node, launch the notebook server: `
Assumes usage of conda / environments

File: getting_started/jupyter_notebooks.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Jupyter Notebooks offer a convenient way to to interactively work on something but can have a heavy computational workload and are thus prohibited from running on the head node. [See the Acceptable Use Policy](../about/accounts.md#acceptable-use-policy). You should run notebooks on the worker nodes instead as described in this tutorial.
+
+For using Jupyter Notebooks you will need to have:
+1. Visual Studio Code installed on your local machine with [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python), [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) and [Remote SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extensions enabled. You should be able to successfully run a python script from within Visual Studio Code on arjuna first.
+2. Jupyter notebook installed in your environment e.g. via a conda environment on arjuna.
+```
+conda create -n my_env
+conda install python
+conda install -c conda-forge notebook
+conda install -c conda-forge nb_conda_kernels
```
Comment (awadell1) [inline getting_started/jupyter_notebooks.md:19]: Drop this, assumes conda has been installed

**Review by awadell1 — CHANGES_REQUESTED** [review]
See comments

**Review by awadell1 — APPROVED** [review]
lgtm

**Pull request merged**